.prettierrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  ...require('gts/.prettierrc.json')
+}
```

Comment:
nit: new line

Reply:
I think I've addressed what you meant (newline at the end of the file) but not sure if that's what you meant?